spec/std/spec/junit_formatter_spec.cr

```diff
@@ -3,14 +3,16 @@ require "xml"
 
 describe "JUnit Formatter" do
   it "reports successful results" do
-    output = build_report do |f|
+    now = Time.utc
+
+    output = build_report_with_mocked_timestamp(now) do |f|
       f.report Spec::Result.new(:success, "should do something", "spec/some_spec.cr", 33, nil, nil)
       f.report Spec::Result.new(:success, "should do something else", "spec/some_spec.cr", 50, nil, nil)
     end
 
     expected = <<-XML
                  <?xml version="1.0"?>
-                 <testsuite tests="2" disabled="0" errors="0" failures="0" time="0.0" hostname="#{System.hostname}">
+                 <testsuite tests="2" disabled="0" errors="0" failures="0" time="0.0" timestamp="#{now.to_rfc3339}" hostname="#{System.hostname}">
```

Comment:
These tests don't really test anything except `build_report_with_mocked_timestamp` - we need a time mocking capability inside the stdlib.

I'd even rather that `build_report_with_mocked_timestamp` was `build_report_with_no_timestamp`, and it just took the timestamp out of the report entirely. At least that way it's obvious it's not being tested.

Reply:
Done.